ppa/archive/models.py

```diff
@@ -136,50 +136,28 @@ class ProtectedFlags(Flags):
     @classmethod
     def deconstruct(cls):
         '''Give Django information needed to make
-        :class:`ProtectedField.no_flags` default in migration.'''
+        :class:`ProtectedWorkFieldFlags.no_flags` default in migration.'''
         # (import path, [args], kwargs)
-        return ('ppa.archive.models.ProtectedFlags', ['no_flags'], {})
+        return ('ppa.archive.models.ProtectedWorkFieldFlags', ['no_flags'], {})
 
-    @classmethod
-    def all_fields(cls):
-        return list(ProtectedFlags.__members__.keys())
+    def __str__(self):
+        return ', '.join(sorted(self.to_simple_str().split('|')))
```

Comment:
I think I would have just done a string replace here, rather than splitting and sorting, since it looks like the flags are returned in the order they are defined on the flag subclass (which is a logical order, since you defined it). But this is fine.